src/components/Tabs/useTabStyles.tsx

```diff
@@ -0,0 +1,37 @@
+import { useTabsContext } from '@reach/tabs';
+import { AbstractButtonProps } from '../AbstractButton';
+
+interface UseTabStylesProps {
+  index: number;
+}
+
+const useTabStyles = ({ index }: UseTabStylesProps): Partial<AbstractButtonProps> => {
+  const { focusedIndex, selectedIndex } = useTabsContext();
+
+  const selectedColor = 'blue-400';
+  const focusedColor = 'navyblue-300';
+
+  const isSelected = selectedIndex === index;
+  const isFocused = focusedIndex === index;
+
+  let borderColor: AbstractButtonProps['borderColor'];
+  if (isSelected) {
+    borderColor = selectedColor;
+  } else if (isFocused) {
+    borderColor = focusedColor;
+  } else {
+    borderColor = 'transparent';
+  }
+
+  return {
```

Comment:
I guess that's intended but IMHO tabs should get visual impact for keyboard navigation.

Reply:
They do!

that's what `isSelected` and `isFocused` handle through the border. 

On tabs where the navigation instantly changes the tab,  you get immediate feedback.

On tabs where it requires pressing "Enter"  to "go" to the tab, then  you get feedback for your selection. 

All that is handled internally through reach-ui and presented through a border. I just didn't want the outline, since there was already another visual indicator present